12.0/apache/entrypoint.sh

```diff
@@ -33,7 +33,9 @@ if version_greater "$installed_version" "$image_version"; then
 fi
 
 if version_greater "$image_version" "$installed_version"; then
+    echo -n "Initializing version $image_version ... "
```

Comment:
the `-n` option, which you use to keep this and `finished.` in one line, does not work if you upgrade from a previous nextcloud version. Because the `occ upgrade` command prints information and is run in betwen this echo and the `finished.` echo.